### ISSUE TITLE
Homepage UX rework: there are too many CTA buttons

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#5155]
Per website UX guidance, a page should ideally have **at most** one primary Call-To-Action (CTA) button, and one or two secondary CTA buttons. Any more and this exponentially dilutes the "call" power of the buttons. 

Our homepage currently has 5 CTA buttons. That is way too many!

> <img width="400" alt="image" src="https://github.com/user-attachments/assets/61d26257-4dd3-41a5-a825-6aff4ee1356c">

We need to rethink and refocus our CTA real estate, and delegate the remaining links to, say, the OTel-highlights ribbon a little further below on the homepage.

References:

- #3306 - had a good intention, but adding a CTA wasn't IMHO the right UX design choice

**Comment by chalin** [comment]
I agree that we could give more prominence to the demo. I'm wondering if the demo is running somewhere, and whether there is a publicly available web interface into at least rudimentary telemetry of the demo? That might be a better "try the demo" experience for a drive-by reader, vs. "install the demo yourself". WDYT?

**Comment by chalin** [comment]
We need to revisit the "Getting started" pages IMHO. At the moment, those pages read more like guiddance on how to read through the rest of the docs. I'd rather see some form of "quick start", but I understand the challenge in setting that up. As I mentioned elsewhere, I think that this would make a good topic of discussion for an upcoming Comms meeting.

**Comment by theletterf** [comment]
Hard agree on revisiting Getting Started. Let's make it a topic for the next call.

**Comment by svrnm** [comment]
Yeah, those getting started guides never turned out to be the way we envisioned them, so maybe we should rethink them and find a different way to get people started.

**Comment by cartermp** [comment]
I think we can axe the dev/ops persona-ish getting started.

**Comment by chalin** [comment]
Closed by:

- #8652
